index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Page Title</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Alegreya SC' rel='stylesheet' />
+  <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
+</head>
+<body>
+  <div id="container">
+    <div id="product-container">
+      <select id="dropdown"></select>
+      <input id="quantity" type="number" name="Quantity" />
+      <input type="submit" value="Add to Cart" />
```

Comment:
These `input`s should be inside a `form` otherwise you cannot listen for a `submit` event and access what the user has chosen to add to cart.